app/models/petition.rb

```diff
@@ -100,6 +102,10 @@ class Petition < ActiveRecord::Base
   scope :with_response, -> { where.not(response_summary: nil, response: nil) }
   scope :with_debate_outcome, -> { joins(:debate_outcome) }
 
+  scope :without_debate_date, -> { where.not(id: DebateOutcome.select(:petition_id).uniq) }
```

Comment:
What's wrong with using `DebateOutcome.select(:petition_id).uniq` here? The scope seems to translate to a good looking SQL with a subquery (and from what I have heard Postgres can optimise it and decide to use a join under the hood if it sees it beneficial?)